editor/editor_file_system.cpp

```diff
@@ -1378,7 +1378,6 @@ void EditorFileSystem::update_script_classes() {
 
 	ScriptServer::save_global_classes();
 	EditorNode::get_editor_data().script_class_save_icon_paths();
-	emit_signal("script_classes_updated");
```

Comment:
Wait, why is this signal emit being removed from here?

Reply:
2 reasons.

1. If the `update_script_classes` method is SUPPOSED to be called every draw call (which is what's currently happening), then we *probably* shouldn't be emitting a signal every frame unnecessarily...?
2. The only reason this signal was added in the first place (by me) was so that the CreateDialog would know the names had been updated and would remove no-longer-existing records from the serialized recent/favorites lists. The change in the PR mentioned in the description, however, removes any need for the signal connection in the first place, therefore the signal emission is no longer necessary either.

Reply:
Oh, which also means I can remove the signal itself! Forgot that bit.

Reply:
Hmm. That's fair, at least by that second reason. I'll put the rest of my response in comment on the PR itself.